extensions/extensions-pipeline-invocation-service/pom.xml

```diff
@@ -83,7 +71,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+            <artifactId>quarkus-resteasy-reactive-jackson</artifactId>
```

Comment:
Talked offline.  It's unclear why this module was using the classic resteasy client instead of this one, but this one works and the other does not.  It seems these modules have had a lot of changes in recent versions of Quarkus.

As for using `extensions-messaging-quarkus`, that module only provides a CDI container config.  Most (all?) Quarkus apps will not rely on this, so it's unclear why this module uses Quarkus at all. It's used in alerting-teams to setup the `TeamsCdiContext`, but that class is unused by the service. I think that this is just unneeded code that could be removed.